README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
+```
+
+Install dependencies
```

Comment:
While 'task dev' is mentioned, it would be useful to describe what the server does or what assets are being built. This gives users an expectation of the outcome.